src/vivarium_public_health/risks/distributions.py

```diff
@@ -294,10 +350,46 @@ def ppf(self, quantiles: pd.Series) -> pd.Series:
 
 
 class DichotomousDistribution(RiskExposureDistribution):
+
     #################
     # Setup methods #
     #################
 
+    def get_exposure_value_columns(self, builder: Builder) -> Optional[List[str]]:
+        if isinstance(self.exposure_data, pd.DataFrame):
+            return builder.data.value_columns()(self.exposure_data)
+        return None
+
+    def get_exposure_data(self, builder: Builder) -> Union[int, float, pd.DataFrame]:
+        exposure_data = super().get_exposure_data(builder)
+
+        if isinstance(exposure_data, (int, float)):
+            return exposure_data
+
+        # rebin exposure categories
+        self.validate_rebin_source(builder, exposure_data)
+        rebin_exposed_categories = set(self.configuration["rebinned_exposed"])
+        if rebin_exposed_categories:
+            exposure_data = self._rebin_exposure_data(exposure_data, rebin_exposed_categories)
+
+        exposure_data = exposure_data[exposure_data["parameter"] == "cat1"]
+        return exposure_data.drop(columns="parameter")
+
+    @staticmethod
+    def _rebin_exposure_data(
+        exposure_data: pd.DataFrame, rebin_exposed_categories: set
+    ) -> pd.DataFrame:
+        exposure_data = exposure_data[
+            exposure_data["parameter"].isin(rebin_exposed_categories)
+        ]
+        exposure_data["parameter"] = "cat1"
```

Comment:
I think this is a fundamental ignorance on my end, but how do you know the parameter is cat1?

Reply:
The rebinned exposure feature allows a user to specify which categories of a polytomous risk they want to consider to be the infected category of a dichotomous risk and "cat1" means infected in that context.